src/popup/ui.js

```diff
@@ -63,3 +78,73 @@ function createButtonsDiv(node, onReloadItems) {
 	buttons.appendChild(archiveButton)
 	return buttons
 }
+
+function loadLabelSelection(article, labels){
+	document.getElementById('content').style = 'display: none;'
+
+	var labelPage = document.getElementById('labelPage')
+	labelPage.style = 'display: flex;'
+
+	var ckBoxContainer = document.getElementById('labelList')
+
+	labels.forEach((item) => {	
+		var li = document.createElement('li');
+
+		var checkbox = document.createElement('input');
+		checkbox.name = "checkboxLabel";
+		checkbox.type = "checkbox";
+		checkbox.value = item.id;
+		checkbox.id = item.id;
+
+		var label = document.createElement('label')
+		label.htmlFor = item.id;
+		label.appendChild(document.createTextNode(' ' + item.name));
+
+		if(article.labels !== null) {
+			article.labels.forEach((gg) => {	
+				if(gg.id == item.id)
+					checkbox.checked = true;
+			})		
+		}
+
+		li.appendChild(checkbox);
+		li.appendChild(label);
+		ckBoxContainer.appendChild(li)
+	})
+
+	var li = document.createElement('li');
+	const backButton = document.createElement('button')
+	backButton.type = 'button'
+	backButton.className = 'closeLabelSelection'
+	backButton.innerHTML = 'Close'
+	backButton.ariaLabel = 'Close'
+	backButton.addEventListener('click', async (event) => {
+		document.getElementById('content').style = 'display: flex;'
+		document.getElementById('labelPage').style = 'display: none;'
+		ckBoxContainer.innerHTML = '';
+	})
+	li.appendChild(backButton)
+
+	const saveButton = document.createElement('button')
+	saveButton.style = 'margin-left: 10px;'
+	saveButton.type = 'button'
+	saveButton.className = 'saveLabelSelection'
+	saveButton.innerHTML = 'Save'
+	saveButton.ariaLabel = 'Save'
```

Comment:
Same as for the `backButton`: Since we already have a proper text content inside the button, we don’t need to use `aria-label` anymore (screen readers can identify the inner content of a button just fine).

```suggestion
	saveButton.innerHTML = 'Save'
```